docs/installing-boot9strap-(mset9-play-store).md

```diff
@@ -22,17 +22,17 @@ On Android phones/tablets, the minimum Android version required is 6.0 (Marshmal
 
 ## What You Need
 
+* The latest release of [MSET9](https://github.com/hacks-guide/MSET9/releases/latest) (the MSET9 `.zip` file)
 * The following applications installed from the Google Play Store:
     * [MSET9 Installer](https://play.google.com/store/apps/details?id=moe.saru.homebrew.console3ds.mset9_installer_android)
     * [ZArchiver](https://play.google.com/store/apps/details?id=ru.zdevs.zarchiver)
     * If you wish, you can sideload these applications instead
-* The latest release of [MSET9](https://github.com/hacks-guide/MSET9/releases/latest) (the MSET9 `.zip` file)
 
 ## Instructions
 
 ### Section I - Prep Work
 
-In this section, you will prepare the SD card data necessary for the MSET9 exploit to trigger.
+In this section, you will prepare the MSET9 exploit by **temporarily** creating a new HOME Menu profile with no user data, and then setting up that profile with only the minimum data required for MSET9 to trigger. Your existing user data will disappear, but will come back when you are finished with this page.
```

Comment:
maybe
``In this section, you will prepare the MSET9 exploit by **temporarily** creating a new HOME Menu profile with almost no user data, and then setting up that profile with only the minimum data required for MSET9 to trigger. Your existing user data will disappear, but will come back when you are finished with this page.``